### DISCUSSION TITLE
Improper chunking for pdf

### DISCUSSION THREAD

**Discussion by anshulgoyal43** [Unstructured-IO/unstructured#3803] [Q&A]
I am trying to do pdf-chunking for my RAG
here is the code I ran

`from unstructured.partition.pdf import partition_pdf

file = "/Users/anshulgoyal/work/pdf_files/a1836-10.pdf"
print("Processing file:", file)

chunks = partition_pdf(filename=file, strategy="fast", chunking_strategy="basic")
for i in chunks:
    print(i)
    print("-"*100)
 `
the link to pdf 'https://www.indiacode.nic.in/bitstream/123456789/18935/1/a1836-10.pdf'

The sentences are broken in middle in chunks, what am I missing?
Is this an issue with pdf or unstructured itself?

**Comment by Sconvai** [comment]
Did you try strategy="hi_res"?
